internal/api/package.go

```diff
@@ -19,7 +20,7 @@ func GetPackageByName(client graphql.Client, orgID string, name string) (*Packag
 	response, err := getPackageByNamingConvention(ctx, client, orgID, name)
 
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error when querying for package %s:\n\t%w", name, err)
```

Comment:
We should probably say something to the effect of please check your project, target and package slugs??